addons/crm/wizard/base_partner_merge.py

```diff
@@ -248,11 +248,18 @@ def write_serializer(item):
             except ValidationError:
                 _logger.info('Skip recursive partner hierarchies for parent_id %s of partner: %s', parent_id, dst_partner.id)
 
-    def _merge(self, partner_ids, dst_partner=None):
+    def _merge(self, partner_ids, dst_partner=None, check_move_lines=True, check_email=True, force_company=False):
         """ private implementation of merge partner
             :param partner_ids : ids of partner to merge
             :param dst_partner : record of destination res.partner
+            :param check_move_lines : multiple journal items allowed ?
```

Comment:
This is absolutely nitpicking but I find the presence of a quotation mark in a doc string so *weird*!